docker/archimedes-base/Makefile

```diff
@@ -1,7 +1 @@
-CERTS ?= ../../development-certs/certs
-baseTag:=$(shell basename "$$(pwd)")
-fullTag:=$(IMAGES_PREFIX)$(baseTag)$(IMAGES_POSTFIX)
-
-image:
-	../build_image Dockerfile $(CERTS) ../../etna/packages/magby
-	if [ -n "$$PUSH_IMAGES" ]; then docker push $(fullTag); fi
```

Comment:
It used to be that every single little image needed all this weird hand written junk just to say "hey build me an image using the Dockerfile in this directory with these dependencies".  Now, just including `stubs.mk` does all this for you with metamagic :mage_man: